W06H01/src/test/PinguFoodLogisticsTest.java

```diff
@@ -23,117 +23,131 @@
 
 
 public class PinguFoodLogisticsTest {
+  private final static String lineSeparator = System.lineSeparator();
   private PrintStream old;
   private ByteArrayOutputStream baos = new ByteArrayOutputStream();
-  
+
   @BeforeEach
-  void ResetRandomSeed() {
+  void resetSeed() {
```

Comment:
I do think renaming and refactoring background code is kind of a waste of time, when the deadline of the task is in two days.

Furthermore, these changes have to be made in every file using this function.